src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx

```diff
@@ -48,6 +48,11 @@ export const ProxyAccount = ({
     addressToShow
   );
 
+  // if proxy type is not in ProxyTypeName enum split camel case string and add spaces
+  const proxyTypeName = ProxyTypeName[proxyType]
```

Comment:
Would be convenient to use a utility function from `entities/proxy/lib`
This might work as well, If `t(...)` returns an empty string
`t(ProxyTypeName[proxyType])  || proxyType.replace(/([a-zA-Z])(?=[A-Z])/g, '$1 ')`

Reply:
Moved it to proxyUtils and moved ProxyTypeName enum there